lib/modules/manager/nuget/update.ts

```diff
@@ -31,27 +31,37 @@ export function bumpPackageVersion(
   try {
     const project = new XmlDocument(content);
     const versionNode = project.descendantWithPath('PropertyGroup.Version')!;
+    const currentProjVersion = versionNode.val;
     const startTagPosition = versionNode.startTagPosition;
-    const versionPosition = content.indexOf(versionNode.val, startTagPosition);
+    const versionPosition = content.indexOf(
+      currentProjVersion,
+      startTagPosition
+    );
 
     const newProjVersion = semver.inc(currentValue, bumpVersion as ReleaseType);
     if (!newProjVersion) {
       throw new Error('semver inc failed');
     }
 
-    logger.debug(`newProjVersion: ${newProjVersion}`);
+    if (currentProjVersion === newProjVersion) {
+      logger.debug('Version was already bumped');
+      return { bumpedContent };
+    }
+
+    logger.debug(
+      `newProjVersion: ${newProjVersion}, project content before bumpVersion(): ${content}`
+    );
+
     bumpedContent = replaceAt(
       content,
       versionPosition,
       currentValue,
       newProjVersion
     );
 
-    if (bumpedContent === content) {
-      logger.debug('Version was already bumped');
-    } else {
-      logger.debug('project version bumped');
-    }
+    logger.debug(
+      `project version bumped, project content after bumpVersion(): ${bumpedContent}`
+    );
```

Comment:
```suggestion
    logger.debug(
      `project version bumped, project content after bumpVersion(): ${bumpedContent}`
    );
```
use original message